nachomemes/bot.py

```diff
@@ -46,19 +46,104 @@
 # recent meme requests (and the resulting meme message)
 RECENT: SimpleCache[int,Message] = SimpleCache(200)
 
-async def report(ctx: Union[Context,Message], ex: Exception, message: str="An error has occured"):
+def print_all_templates(config: GuildConfig) -> dict:
+    memes = STORE.list_memes(config.guild_id, ("name", "description"))
+    return {"embed": Embed(
+        title="Available Templates",
+        description = "".join((f"\n{m['name']}: *{m['description']}*" for m in memes))
```

Comment:
I think there's an extra set of parenthesis here.

Reply:
there is not.  the outer parens are for the function call, the inner are for the generator expression